### PR TITLE
dataBackupConfigured flag

### PR DESCRIPTION
**Pull request by JesseGoodier**

### CODE REVIEW

File: cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml [hunk 1]
```diff
@@ -1055,7 +1055,8 @@ data:
             add_header 'Access-Control-Allow-Origin' '*' always;
             add_header 'Access-Control-Allow-Methods' 'GET, PUT, POST, DELETE, OPTIONS' always;
             return 200 '\n
-                {"ssoConfigured": "{{ template "ssoEnabled" . }}"}\n
+                {"ssoConfigured": "{{ template "ssoEnabled" . }}"},\n
+                {"dataBackupConfigured": "{{ template "dataBackupConfigured" . }}"}\n
```
Comment (thomasvn) [inline cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml]:
```suggestion
                {
                "ssoConfigured": "{{ template "ssoEnabled" . }}",
                "dataBackupConfigured": "{{ template "dataBackupConfigured" . }}"
                }
```

I think (?) we should be returning a single JSON object and not two?

**Comment by thomasvn** [comment]
Looks good! But in what scenarios in 2.x is `etlBucketConfigSecret` configured and `federatedStorageConfigSecret` not?

**Comment by jessegoodier** [comment]
> Looks good! But in what scenarios in 2.x is `etlBucketConfigSecret` configured and `federatedStorageConfigSecret` not?

single cluster

**Review by thomasvn — APPROVED** [review]
Once merging, we should check that any frontend calls to `/model/productConfigs` are still able to properly parse the response being returned by Nginx.

I'm not 100% certain that the formatting here is correct

**Comment by thomasvn** [comment]
Looking at the nightly environment, it seems like calls to `/model/productConfigs` show up like this:

```


            {
            "ssoConfigured": "false",
            "dataBackupConfigured": "true"
            }
        
```

The extra whitespace is weird, but I _think_ this is still valid parseable JSON. We may need to revisit to clean up one day.

**Pull request merged**